src/calculateRentalCost.js

```diff
@@ -1,10 +1,16 @@
-/**
- * @param {number} days
- *
- * @return {number}
- */
 function calculateRentalCost(days) {
```

Comment:
The function name `calculateRentalCost` is descriptive and clearly indicates its purpose, which is good practice as per the checklist item 1.